.github/workflows/publish_imgs.yml

```diff
@@ -84,7 +84,6 @@ jobs:
       - name: Authenticate and Execute Publish Script
         id: publish
         run: |
-          set -x
```

Comment:
Removing this since I think it was only here for troubleshooting

Reply:
Yeah this was mostly to help with debugging. If anything, we could comment it out, but I think it's fine to remove it from the yml itself.